.github/CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ Available categories include:
 * `PennyLane Help`: For help and advice using PennyLane
 * `PennyLane Development`: For discussion of PennyLane development
 * `PennyLane Plugins`: For discussion of the available PennyLane plugins, and plugin development
-* `Xanadu Software`: For discussion relating to other Xanadu software projects, including [StrawberryFields](https://github.com/xanaduai/strawberryfields), [QMLT](https://github.com/xanaduai/hafnian), [Hafnian](https://github.com/xanaduai/hafnian), and [OpenBoson](https://github.com/xanaduai/sfopenboson).
+* `Xanadu Software`: For discussion relating to other Xanadu software projects, including [StrawberryFields](https://github.com/xanaduai/strawberryfields), [QMLT](https://github.com/XanaduAI/QMLT), [TheWalrus](https://github.com/xanaduai/thewalrus), and [OpenBoson](https://github.com/xanaduai/sfopenboson).
```

Comment:
Since QMLT is deprecated, let's remove it from this list :slightly_smiling_face:

Reply:
```suggestion
* `Xanadu Software`: For discussion relating to other Xanadu software projects, including [StrawberryFields](https://github.com/xanaduai/strawberryfields), [TheWalrus](https://github.com/xanaduai/thewalrus), and [OpenBoson](https://github.com/xanaduai/sfopenboson).
```